elasticache/redis/Acornfile

```diff
@@ -0,0 +1,88 @@
+name:        "AWS Elasticache Redis"
+description: "Amazon's Redis compatible Elasticache cluster"
+
+args: {
+	// Name assigned to the cluster during creation. Default value is "RedisCluster".
+	clusterName: "RedisCluster"
+
+	// Key value pairs to apply to all resources.
+	tags: {}
+
+	// Deletion protection. Must be set to false in order to delete the Redis clutser. Default value is false.
+	deletionProtection: false
+}
+
+services: redis: {
+	name: "Redis Admin"
+	generated: job: "apply"
+}
+
+jobs: apply: {
+	build: {
+		context:    "../"
+		dockerfile: "../redis.Dockerfile"
+		additionalContexts: {
+			common: "../../libs"
+			utils:  "../../utils"
```

Comment:
I don't know that this is needed any more. At least not for scripts.

Reply:
Removed in the latest commit.